documentation/sdk/docs/how_tos/native_tokens/burn.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Burn Native Tokens
+sidebar_label: Burn
+description: 'How to burn native tokens'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- burn
+- burn native token
+- foundry
+- nodejs
+- python
+- rust
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import NodejsCode from '!!raw-loader!../../../../../bindings/nodejs/examples/how_tos/native_tokens/burn.ts';
+import PythonCode from '!!raw-loader!../../../../../bindings/python/examples/how_tos/native_tokens/burn.py';
+import RustCode from '!!raw-loader!../../../../../sdk/examples/how_tos/native_tokens/burn.rs';
+
+You may want to burn some of your circulating native tokens. To do so, you will need to call the
+`Account.burn(burn, options)` function.
+
+:::warning
+
+If you burn your native tokens, you will not be able to destroy the foundry that created them. If you want to destroy
```

Comment:
Since the native tokens could also be minted by someone else, it should maybe just say
```suggestion
If you burn native tokens, it will be impossible to destroy the foundry that created them and get its storage deposit amount back. If you want to destroy
```

Reply:
Sounds good